src/main/java/mousio/etcd4j/EtcdClient.java

```diff
@@ -92,10 +95,10 @@ public EtcdClient(SslContext sslContext, URI... baseUri) {
    */
   public EtcdClient(EtcdSecurityContext securityContext, URI... baseUri) {
     this(new EtcdNettyClient(
-      securityContext,
-      (baseUri.length == 0)
-        ? new URI[] { URI.create("https://127.0.0.1:4001") }
-        : baseUri
+            securityContext,
+            (baseUri.length == 0)
+                    ? new URI[] { URI.create("https://127.0.0.1:4001") }
+                    : baseUri
```

Comment:
Please do no change the format of pre-existing code

Reply:
Sorry about that! I'll fix it asap